lib/sycamore/sycamore/transforms/merge_elements.py

```diff
@@ -611,6 +611,195 @@ def postprocess_element(self, elem: Element) -> Element:
+            or element1.type is None
+            or element2.type is None
+        ):
+            return False
+
+        # DO NOT MERGE across pages
+        if (
+            not self.merge_across_pages
+            and element1.properties[DocumentPropertyTypes.PAGE_NUMBER]
+            != element2.properties[DocumentPropertyTypes.PAGE_NUMBER]
+        ):
+            return False
+
+        if element1.data["token_count"] + 1 + element2.data["token_count"] > self.max_tokens and element2.type not in [
+            "Section-header",
+            "Title",
+        ]:
+            # Add header to next element
+            element2["_header"] = element1["_header"]
+            if element1.data["_header"]:
+                if element2.text_representation:
+                    element2.text_representation = element1.data["_header"] + "\n" + element2.text_representation
+                else:
+                    element2.text_representation = element1.data["_header"]
+            return False
+
+        # Merge consecutive section headers/titles and save as a section-header element
+        if element1.type in ["Section-header", "Title"] and element2.type in ["Section-header", "Title"]:
+            return True
+
+        # MERGE adjacent 'text' elements
+        small = {"Text", "List-item", "Caption", "Footnote", "Formula", "Page-footer", "Page-header"}
+        if (element1 is not None) and (element1.type in small) and (element2 is not None) and (element2.type in small):
+            return True
+
+        # Add header to next element
+        if element2.type not in ["Section-header", "Title"]:
+            element2.data["_header"] = element1.data["_header"]
+            if element2.text_representation:
+                if element2.data["_header"]:
+                    element2.text_representation = element2.data["_header"] + "\n" + element2.text_representation
+            else:
+                element2.text_representation = element2.data["_header"]
+        return False
+
+    def merge(self, elt1: Element, elt2: Element) -> Element:
+        """Merge two elements; the new element's fields will be set as:
+            - type: "Section-header" , "MergedText"
+            - binary_representation: elt1.binary_representation + elt2.binary_representation
+            - text_representation: elt1.text_representation + elt2.text_representation
+            - bbox: the minimal bbox that contains both elt1's and elt2's bboxes
+            - properties: elt1's properties + any of elt2's properties that are not in elt1
+            note: if elt1 and elt2 have different values for the same property, we take elt1's value
+            note: if any input field is None we take the other element's field without merge logic
+
+        Args:
+            element1 (Tuple[Element, int]): the first element (and number of tokens in it)
+            element2 (Tuple[Element, int]): the second element (and number of tokens in it)
```

Comment:
I know that this is what the other mergers say, but can you change this to the below?
```
Args:
    element1 (Element): the first element (numbers of tokens in it is stored by `preprocess_element` as element1["token_count"])
    element2 (Element): the first element (numbers of tokens in it is stored by `preprocess_element` as element2["token_count"])
```